### ISSUE TITLE
the NB list page at `<server url>/<user>` should show previous notebook revision and allow them to be opened

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#793]
from initial user stories at:
https://docs.google.com/document/d/1nq2mY3L57Wzfno5xj_eVcMECppB7iPA7jdyG5jRIUr4/edit?ts=5b7c6424
assigning @wlach initially, but just so that he can review, update, and assign to others as needed

**Comment by bcolloran** [comment]
maybe blocked by #792

**Comment by bcolloran** [comment]
this may require some thought around branching histories -- i.e., what happens when a user opens an old NB revision and then tries to save that? as a simple-ish MVP perhaps initially we could require a "save as" type action that would require a rename/starting a new version history chain, until we work up to supporting a proper revision tree/graph

**Comment by wlach** [comment]
I think it might make more sense to allow viewing the revision list inside the notebook itself, though perhaps we could add some kind of shortcut to that from the notebook list.

**Comment by bcolloran** [comment]
+1 i think that's the interface i'd prefer ultimately as well. i filed this issue thinking that it might faster to implement on the landing page, but we can change the description

**Comment by hamilton** [comment]
@bcolloran As of now, `/notebooks/<notebook-id>/revisions/` will give you a list of the revisions of a notebook, and allow you to view them at `/notebooks/<notebook-id>/?revision=<revision-id>`. Would this include viewing the revisions list in a modal? If so, I can add add this modal to the client-side code.

**Comment by bcolloran** [comment]
we have that as a separate user story in #873, but it's lower priority since there are still a good number of basic workflow things to get right. i think you have closed this specific issue with one of your PRs today, no?

**Comment by hamilton** [comment]
Ah, yes. This was fixed with #860.
